### PR TITLE
Hexvexed recipe 2 :/

### PR DESCRIPTION
**Pull request by punsith**

### CODE REVIEW

File: haiku-games/hexvexed/hexvexed-0.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,45 @@
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION="HexVexed is a maddening addictive and easy to play puzzle game. \
+Arrange hexagonal tiles into a grid so that the numbers and colors match \
+where they touch. This game is a variation based on DarkWyrm's BeVexed, but instead \
+of squares, the tiles are hexagon shaped. It's 1.5x the fun!"
+HOMEPAGE="https://bitbucket.org/scottmc/hexvexed"
+COPYRIGHT="2009-2016 Scott McCreary"
+LICENSE="MIT"
+REVISION="1"
+commit="68437a22c788"
```
Comment (fbrosson) [inline haiku-games/hexvexed/hexvexed-0.0.1.recipe]: Can you please replace
```
commit="68437a22c788"
```
by
```
commit="f9cd2a7b1ba1"
```

File: haiku-games/hexvexed/hexvexed-0.0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/$commit.zip"
+CHECKSUM_SHA256="35e0904d91f15fefe801cdb7b9f9bdf4dc925de69f5b8818bec24cc4c22dc6d7"
```
Comment (fbrosson) [inline haiku-games/hexvexed/hexvexed-0.0.1.recipe]: Can you please replace
```
SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/$commit.zip"
CHECKSUM_SHA256="35e0904d91f15fefe801cdb7b9f9bdf4dc925de69f5b8818bec24cc4c22dc6d7"
```
by
```
SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/$commit.tar.bz2"
CHECKSUM_SHA256="72b1a5e2d3d91a453b289948edfcb437d8c2e7f8698255a109f57e6aad6a4138"
```

File: haiku-games/hexvexed/hexvexed-0.0.1.recipe [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+SOURCE_DIR="scottmc-hexvexed-$commit"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```
Comment (fbrosson) [inline haiku-games/hexvexed/hexvexed-0.0.1.recipe]: Can you please replace
```
ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```
by
```
ARCHITECTURES="?x86_gcc2 !x86 !x86_64"
```

**Comment by punsith** [comment]
@fbrosson  sure I will change it :D

**Comment by fbrosson** [comment]
Cool, thanks @punsith!

BTW, I cannot reopen this PR. The "Reopen and comment" button is disabled.
Can you try to reopen it yourself?

**Comment by fbrosson** [comment]
@scottmc, shall we merge the recipe with the current tags (that @punsith has kindly added on my request) and  that say that the recipe does not build on x86 and x86_64 and that it is untested on x86_gcc2?
In other words, did someone manage to launch HexVexed on x86_gcc2 without crashing it?

**Comment by punsith** [comment]
@fbrosson  you are welcome

**Pull request merged**